VTEX - Antifraud Provider API.json

```diff
@@ -1395,48 +976,30 @@
                     "OAuth Flow"
                 ],
                 "summary": "1. Retrieve Token",
-                "description": "This endpoint is used to retrieve a payment provider token.",
+                "description": "This endpoint is used to retrieve a payment provider token.\r\n\r\n## Permissions\r\n\r\nThis endpoint does not require [authentication](https:\/\/developers.vtex.com\/docs\/guides\/authentication) or [permissions](https:\/\/help.vtex.com\/en\/tutorial\/license-manager-resources--3q6ztrC8YynQf6rdc6euk3).",
```

Comment:
Se esse endpoint não requer authentication, por que tem security nele?

Reply:
Adicionado nova descrição, conforme conversamos: ## Permissions\r\n\r\nThis endpoint does not require [permissions](https:\/\/help.vtex.com\/en\/tutorial\/license-manager-resources--3q6ztrC8YynQf6rdc6euk3).